front/cypress/e2e/project_description_builder/project_description_builder_navigation.cy.ts

```diff
@@ -5,7 +5,6 @@ describe('Project description builder navigation', () => {
   const projectTitle = randomString();
 
   before(() => {
-    cy.setAdminLoginCookie();
     cy.getAuthUser().then((user) => {
```

Comment:
Why can we remove this _setAdminLoginCookie_? Do we know that when this before runs we already have admin credentials? :thinking:

Reply:
> cy.getAuthUser signs in as an admin.

Ah ok - wasn't aware! ^